sample-app/src/components/VisualAutocomplete/VisualAutocompleteSection.tsx

```diff
@@ -0,0 +1,17 @@
+import { Result } from '@yext/answers-headless-react';
+
+export interface VisualAutocompleteSectionProps {
+  verticalKey: string,
+  children: (results: Result[]) => JSX.Element,
+  limit?: number
+}
+
+/**
+ * VisualAutocompleteSection is responsible for providing configuration to VisualAutocompleteEntities,
+ * and allowing users to specify arbitrary VisualAutocomplete layouts.
+ * 
+ * This is done by providing results through props.children, using an FACC (function as a child component).
+ */
+export function VisualAutocompleteSection(_: VisualAutocompleteSectionProps) {
```

Comment:
I'm not totally sold on the purpose of `VisualAutocompleteSection`. It seems like not much more than a pass-through to `VisualAutocompleteEntities`. Do you anticipate it will do more in the future?